TODO.md

```diff
@@ -16,20 +16,26 @@
     * Load balancers
     * NAT
 * Top
+  * Offline
+    * Problems loading reference 'https://schemastore.azurewebsites.net/schemas/json/tsconfig.json': Unable to load schema from 'https://schemastore.azurewebsites.net/schemas/json/tsconfig.json': Unable to connect to https://schemastore.azurewebsites.net/schemas/json/tsconfig.json. Error: getaddrinfo ENOTFOUND schemastore.azurewebsites.net schemastore.azurewebsites.net:443
   * Cleanup
+    * Why t.record(t.string, t.any) vs t.record(t.string, t.string)?
+    * Remove "as NodeSpec[]" from azure_convert_parity.test.ts. Use "const expected: NodeSpec[] = ..."
```

Comment:
Agree with removing the as. I'll cleaned up a bit already. I'll sweep the code for other places this is happening

Reply:
I tried making this change, but it proved to be too complex, so I thought I'd leave it to a later commit. The problem is that expected isn't really a NodeSpec[] because each rule contains `id` and `source`. Therefore, this is first a type declaration problem in RuleSpec and ForwardRuleSpec and NodeSpec. Once that problem has been resolved, we can fix your unit test.